src/test/java/com/exasol/adapter/dialects/elasticsearch/ColumnNameMapperTest.java

```diff
@@ -0,0 +1,35 @@
+package com.exasol.adapter.dialects.elasticsearch;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.stream.Stream;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class ColumnNameMapperTest {
+    /**
+     * @return A stream of arguments with the ElasticSearch column name representation on the left, and Exasol's on the
+     *         right.
+     */
+    static Stream<Arguments> mappedColumnNames() {
+        return Stream.of(//
+                Arguments.of("book", "book"), //
+                Arguments.of("book.author", "book/author"), //
+                Arguments.of("book.author.name", "book/author/name") //
+        );
+    }
```

Comment:
Use `CsvSource` for better readability.

Reply:
Ah, I just saw that you use the same data in reverse direction. In that case, the method source is OK.

Reply:
Also test some bad-weather scenarios please!